suite/tests/client-interface/drx-scattergather.c

```diff
@@ -284,6 +321,11 @@ test_avx2_avx512_scatter_gather(void)
                             ref_idx64_val64_xmm_ymm_zmm, test_idx64_vec,
                             output_xmm_ymm_zmm))
         return false;
+    /* As pointed out above, the scatter tests scatter data from ref_xmm_ymm_zmm into the
+     * array output_sparse_test_buf. Its the inverse of the gather test, so the source
+     * data for the xmm, ymm, and zmm scatter instruction is concatenated in
```

Comment:
s/instruction is/instructions are/ OR s/the/each/

Reply:
Done.